libvcx/src/aries/handlers/connection/connection.rs

```diff
@@ -72,9 +78,11 @@ impl Connection {
     pub fn create(source_id: &str, autohop: bool) -> VcxResult<Connection> {
         trace!("Connection::create >>> source_id: {}", source_id);
         let pairwise_info = PairwiseInfo::create()?;
+        let connection_sm = SmConnection::Inviter(SmConnectionInviter::new(source_id, pairwise_info, send_message));
```

Comment:
I guess we will forever disagree on this 😄

Reply:
In general I do like more lines rather than too long / complicated / nested expression. 

Inlined here though

Reply:
At least we agree to disagree :) It's clearly a matter of subjective preference